frame/transaction-fee/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 //! Loosely based on https://github.com/paritytech/substrate/blob/master/frame/transaction-payment/src/lib.rs
 //! but with added support for `MultiCurrency` using a `Dex` interface.
 
+#![cfg_attr(not(test), warn(clippy::disallowed_method, clippy::indexing_slicing))] // allow in tests
```

Comment:
we put this into each pallet in frame. is it possible to put this rule on folder/directory?

Reply:
Unfortunately not, lints need to be specified per crate unless you specify them on the command line (i.e. `cargo clippy -- -W clippy::disallowed_method`)